eng/scripts/Language-Settings.ps1

```diff
@@ -5,6 +5,48 @@ $packagePattern = "*.zip"
 $MetadataUri = "https://raw.githubusercontent.com/Azure/azure-sdk/master/_data/releases/latest/python-packages.csv"
 $BlobStorageUrl = "https://azuresdkdocs.blob.core.windows.net/%24web?restype=container&comp=list&prefix=python%2F&delimiter=%2F"
 
+function Get-AllPackageInfoFromRepo
+{
+  $allPackageProps = @()
+  Push-Location $RepoRoot
+  $allSetupProps = $null
+  try{
+    pip install packaging==20.4 -q -I
+    $allSetupProps = (python -c "import sys; import os; sys.path.append(os.path.join('eng', 'scripts')); \
```

Comment:
Why isn't this just `python get_package_properties.py $rootPath`?

Reply:
Am trying to capture the output back in PowerShell. I don't think that works.

Reply:
I would like to better understand why that wouldn't work as it should be like any other command line tool we are run we will get the output from the console.